test/cpp/jit/test_constant_pooling.cpp

```diff
@@ -93,7 +93,7 @@ graph():
 }
 
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
-TEST(constantPoolingTest, DictConstantPooling) {
+TEST(constantPoolingTest, DictconstantPooling) {
```

Comment:
Accident? It seems DictConstantPooling is correct CamelCase.

Reply:
Good catch!